src/main/resources/assets/botania/lang/zh_cn.lang

```diff
@@ -2129,14 +2129,14 @@ botania.page.distributor1=合成分配器
 # -- MANA VOID
 botania.entry.manaVoid=魔力销毁器
 botania.tagline.manaVoid=销毁一切进来的魔力
-botania.page.manaVoid0=&1魔力销毁器&0通俗的说,就是&4魔&0 &4力&0的垃圾桶.所有进入这个方 块的&4魔力&0都会消失在虚空,再 也不会出现.<br>将它放在一个&1魔力池&0下方会 让池子总是接收&4魔力&0,但销毁 超出容量的部分.
+botania.page.manaVoid0=简单来说，&1魔力销毁器&0是&4魔力&0 的垃圾桶。所有进入这个方块的 &4魔力&0都会被虚空永远地吞噬。<br>在&1魔力池&0下方放一个这个，就 可以让池子满地时候销毁多余 的&4魔力&0。
 # No translation # botania.page.manaVoid1=Even Chad from accounting?
 
 # -- MANA FLUXFIELD
 botania.entry.rfGenerator=魔力转换器
-botania.tagline.rfGenerator=魔力变RF啦!
-botania.page.rfGenerator0=将&4魔力&0引入红石能量场,就可 以将其从自然能量源形态转 为&4红石流量&0.<br>将一魔力发射器对向此方块, 就可把进入其中的&4魔力&0转化 为&4RF&0.之后就能用各种传输&4RF&0 的装置来输出.
-# No translation # botania.page.rfGenerator1=The bullet guys. I bit it.
+botania.tagline.rfGenerator=将 Mana 转化为能量
+botania.page.rfGenerator0=将&4魔力&0引入红石能量场,就可 以将其从自然能量源形态转 为&4能量&0。<br>只需将魔力发射器对准这转 换器，转换器就可以把输入的&4魔力&0转化 为&4能量&0，并输出到邻近的能量传输设备中。
+botania.page.rfGenerator1=传统的挑战者
```

Comment:
I wanted to put *Goliath* on my first glance but dropped that idea later. Id est, how about just delete this block? 

> Also, [do quotation properly][citation-1] please

[citation-1]: https://github.com/CovertJaguar/Railcraft/issues/1097#issuecomment-284556917